### PR TITLE
other/bugfixes: commented out unfinished features & fixed wars

### PR DESCRIPTION
**Pull request by Ryos00**
- commented out exile
- commented out map
- commented out Cats Outside The Clan
- removed numpy and noise from requirements
- fixed wars and clan relations junk

### CODE REVIEW

**Comment by Ryos00** [comment]
pretend u don't see all those commits of me failing to fix relations

**Review by sedgestripe — APPROVED** [review]

File: scripts/events.py [hunk 1]
```diff
@@ -156,11 +156,10 @@ def check_clan_relations(self):
                     other_clan.relations = 10
                 else:
                     self.at_war = False
-                    r_num = choice([-1, 1])
-                    other_clan.relations = str(
-                        int(other_clan.relations) + r_num)
+                    other_clan.relations = other_clan.relations
```
Comment (Lixxis) [inline scripts/events.py:159]: This line is irreleavant

File: scripts/events.py [hunk 2]
```diff
@@ -156,11 +156,10 @@ def check_clan_relations(self):
...
             if war_notice:
                 game.cur_events_list.append(war_notice)
+            print(other_clan.relations)
```
Comment (Lixxis) [inline scripts/events.py:162]: Maybe if it is not needed anymore, you could delete this line

**Review by Lixxis — CHANGES_REQUESTED** [review]

**Review by Lixxis — APPROVED** [review]

File: scripts/patrol.py [hunk 3]
```diff
@@ -800,11 +801,100 @@ def add_possible_patrols(self):
                             10,
                             win_skills=['great hunter', 'fantastic hunter'])
             ])
-
+        
 
         # other_clan patrols
-        if len(game.clan.all_clans) > 0:
-            1 == 1  # will add here
+        if randint(1, 1) == 1 and self.other_clan is not None:
```
Comment (Dpeta) [inline scripts/patrol.py:807]:
```suggestion
        if self.other_clan is not None:
```

Comment (Dpeta) [reply scripts/patrol.py:807]: (Not a full review btw, just noticed that xd)

**Review by Thlumyn — APPROVED** [review]

**Pull request merged**